docs/source/tutorial_crowdsourcing.md

```diff
@@ -0,0 +1,255 @@
+    
+To run a crowdsourcing task, launch its run file (typically `run.py`) with the proper flags, using a command like the following:
+
+```bash
+python run.py \
+mephisto.blueprint.num_conversations <num_conversations> \
+mephisto.task.task_reward <reward>
+```
+(Note that the command will launch HITs on the sandbox by default.) For instance, to create 2 conversations for the [QA Data
+Collection](https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing/tasks/qa_data_collection)
+task with a reward of $0.05 per assignment in sandbox mode, run:
+
+```bash
+python parlai/crowdsourcing/tasks/qa_data_collection/run.py \
+mephisto.blueprint.num_conversations 2 \
+mephisto.task.task_reward 0.05
+```
+
+Make sure to test your task in sandbox mode first before pushing it live: see the [crowdsourcing README](https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing#running-tasks-live) for how to run a live task.
+
+Additional flags can be used for more specific purposes:
+
+-   `mephisto.task.maximum_units_per_worker` ensures that a single Turker is only able to complete one
+    assignment, thus ensuring that each assignment is completed by a different
+    person.
+-   `mephisto.task.allowed_concurrent` prevents a Turker from entering more
+    than a certain number of conversations at once (by using multiple
+    windows/tabs). This defaults to 0, which is unlimited.
+-   `mephisto.task.assignment_duration_in_seconds` sets a maximum limit for how long a specific worker can work on your task.
+
+See the [crowdsourcing README](https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing#mturk-specific-task-configuration) for some more commonly used command-line flags.
+
+Reviewing Turker's Work
+-----------------------
+
+You can programmatically review work using the commands available in the
+`CrowdTaskWorld` class: for example, see the sample code in the docstring of [`CrowdTaskWorld.review_work()`](https://github.com/facebookresearch/ParlAI/blob/master/parlai/crowdsourcing/utils/worlds.py). In particular, you can set HITs to be
+automatically approved if they are deemed completed by the world.
+
+By default, if you don't take any action to approve/reject HITs within 1 week, all HITs will be auto-approved
+and Turkers will be paid.
+
+Mephisto MTurk Tips and Tricks
+----------------------------
+
+### Approving Work
+
+-   Unless you explicitly set the `auto_approve_delay` argument in [`create_hit_type()`](https://github.com/facebookresearch/Mephisto/blob/master/mephisto/abstractions/providers/mturk/mturk_utils.py), or approve work by calling [`MTurkAgent.approve_work()`](https://github.com/facebookresearch/Mephisto/blob/master/mephisto/abstractions/providers/mturk/mturk_agent.py), work will be auto-approved after 7 days. Workers like getting paid quickly, so be mindful to not have too much delay before their HITs are approved.
```

Comment:
i'd need to update mephisto code to set the auto_approve_delay? i.e. I cannot set it via cmd line or in a conf file?

Reply:
I think that's right, but perhaps @JackUrb can confirm

Reply:
At the moment this is correct - though we're getting to the point where this may be exposed as a hydra argument soon. I have a backlog of hydra args to get to, but have been waiting on finishing https://github.com/facebookresearch/Mephisto/pull/224 to have the functionality blocking one of the major ones.